internal/proxy/task.go

```diff
@@ -1183,7 +1184,15 @@ func (cct *createCollectionTask) PreExecute(ctx context.Context) error {
 
 func (cct *createCollectionTask) Execute(ctx context.Context) error {
 	var err error
+	logCtx := logutil.WithReqID(cct.ctx, cct.GetBase().GetMsgID())
+
+	logutil.Logger(logCtx).Debug("send create collection request to RootCoord")
 	cct.result, err = cct.rootCoord.CreateCollection(ctx, cct.CreateCollectionRequest)
+	logutil.Logger(logCtx).Debug("receive create collection response from RootCoord",
```

Comment:
Which context shall we use in line 1190, which invokes `CreateCollection` of rootcoord?